rustls/Cargo.toml

```diff
@@ -40,7 +40,7 @@ brotli-decompressor = { workspace = true, optional = true }
 hashbrown = { workspace = true, optional = true }
 log = { workspace = true, optional = true }
 # only required for no-std
-once_cell = { version = "1.16", default-features = false, features = ["alloc", "race"] }
+once_cell = { workspace = true, default-features = false, features = ["alloc", "race"] }
```

Comment:
I think the `, default-features = false, features = ["alloc", "race"] ` can be dropped here?

Reply:
done